README.md

```diff
@@ -59,6 +62,12 @@ To create a cluster from Kubernetes source:
 Multi-node clusters and other advanced features may be configured with a config
 file, for more usage see [the docs][user guide] or run `kind [command] --help`
 
+To make `kubectl` use your local kind cluster, use:
```

Comment:
We already print a per-platform relevant usage when you run kind create cluster. I'd rather not duplicate that here. For example this version is now lacking windows.

Reply:
I probably missed that when I first created a cluster... But you have a point, closing this then!

Reply:
Thanks anyhow though, I appreciate documentation efforts especially :-)